templates/listing_table.tpl

```diff
@@ -6,39 +6,54 @@
         <th class="hidden-print">
-            <a href="index.php?page=display&dn={$entry.dn|escape:'url'}&search={$search}" class="btn btn-info btn-sm{if $listing_linkto===false} hidden{/if}" role="button" title="{$msg_displayentry}">
+            <a href="index.php?page=display&dn={$entry.dn|escape:'url'}&search={$search}"
+                class="btn btn-info btn-sm{if $listing_linkto===false} hidden{/if}" role="button"
+                title="{$msg_displayentry}">
                 <i class="fa fa-fw fa-id-card"></i>
             </a>
-        {if $display_unlock_button}
-            <a href="index.php?page=unlockaccount&dn={$entry.dn|escape:'url'}&returnto=searchlocked" class="btn btn-success btn-sm" role="button" title="{$msg_unlockaccount}">
+            {if $display_unlock_button}
+            <a href="index.php?page=unlockaccount&dn={$entry.dn|escape:'url'}&returnto=searchlocked"
+                class="btn btn-success btn-sm" role="button" title="{$msg_unlockaccount}">
                 <i class="fa fa-fw fa-unlock"></i>
             </a>
-        {/if}
+            {/if}
         </th>
-    {foreach $listing_columns as $column}
+        {foreach $listing_columns as $column}
         <td>
-        {$attribute=$attributes_map.{$column}.attribute}
-        {if ({$entry.$attribute.0})}
+            {if $display == "search"}
+            {$attribute=$attributes_map.{$column}.attribute}
+            {if ({$entry.$attribute.0})}
             {if $listing_linkto|is_array && in_array($column, $listing_linkto)}
-                 <a href="index.php?page=display&dn={$entry.dn|escape:'url'}&search={$search}" title="{$msg_displayentry}">
-            {/if}
-            {foreach $entry.{$attribute} as $value}
+            <a href="index.php?page=display&dn={$entry.dn|escape:'url'}&search={$search}" title="{$msg_displayentry}">
+                {/if}
+                {foreach $entry.{$attribute} as $value}
                 {if $value@index eq 0}{continue}{/if}
                 {$type=$attributes_map.{$column}.type}
                 {include 'value_displayer.tpl' value=$value type=$type}
-            {/foreach}
-            {if $listing_linkto|is_array && in_array($column, $listing_linkto)}
-                 </a>
+                {/foreach}
+                {if $listing_linkto|is_array && in_array($column, $listing_linkto)}
+            </a>
             {/if}
-        {else}
+            {else}
             {if $show_undef}<i>{$msg_notdefined}</i>{else}&nbsp;{/if}
-        {/if}
+            {/if}
+            {/if}
+            {if $display == "audit"}
+            {if $column == "result" or $column == "action"}
+            {$msg_{$entry.$column}}
+            {elseif $column == "dn"}
+            <a href="index.php?page=display&dn={$entry.dn|escape:'url'}&search={$search}" title="{$msg_displayentry}"\>
+            {{$entry.$column|regex_replace:"/(cn=)/":""}|regex_replace:"/(,ou=.*)/":""}
```

Comment:
You should use: 

{include 'value_displayer.tpl' value={$entry.$column} type="dn_link"}